connector.go

```diff
@@ -116,11 +118,18 @@ func newConnector(cf any, mapper Mapper, logger logger.Logger, errorLogger logge
 	dcpConfig.Checkpoint.Type = "manual"
 
 	connector.dcp = dcp
-	connector.bulk, err = bulk.NewBulk(
+
+	v7Client, err := v7.New(cfg)
```

Comment:
Better naming maybe from v7Client to esClient

Reply:
it was esClient but I thought that since the package name starts with elasticsearch we don't need to add es prefix. maybe we can simply say client so that we can use it like elasticsearch.Client